### PR TITLE
Porting over small changes/fixes from Bronzes

### PR DESCRIPTION
**Pull request by geealbers**
A handful of small changes/fixes coming out of *Bronze Guidelines*.

One includes allowing users to specify `target` in markdown links: `[lnk](url){target=_blank}`. Quire's default behavior is that internal links open in the same window, and external in a new window. I think that's good, and should be left as is, but for *Bronzes* I wanted a number of specific internal links to open in new windows and based on past community feedback I thought that others might have the same need in their projects. Certainly open for discussion though if any of you see a downside to enabling this.

### CODE REVIEW

**Review by mphstudios — CHANGES_REQUESTED** [review]
See suggestions inline.

File: packages/11ty/.gitignore [hunk 1]
```diff
@@ -19,6 +19,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# macOS
+.DS_Store
+
```
Comment (mphstudios) [inline packages/11ty/.gitignore:24]: @hbalenda may have addressed adding `.DS_Store` to the ignore patterns in a separate pull-request.

File: packages/11ty/_includes/components/contributor/bio.js [hunk 2]
```diff
@@ -47,11 +47,11 @@ module.exports = function (eleventyConfig) {
...
     const contributorPages = pages.map(({ data, url }) => {
       const { label, subtitle, title } = data
-      return `${link({
+      return `<li>${link({
         classes: ['quire-contributor__page-link'],
         name: pageTitle({ label, subtitle, title }),
         url,
-      })}`
+      })}</li>`
     })
```
Comment (mphstudios) [inline packages/11ty/_includes/components/contributor/bio.js:55]: To keep the list markup as a concern of the method, we might rewrite this as
```
const contributorPagesList = {
  const items = pages.map(({ data, url }) => {
    const { label, subtitle, title } = data
    const classes = ['quire-contributor__page-link']
    const name = pageTitle({ label, subtitle, title })
    return `<li>${link({ classes, name, url })}</li>`
  })
  return html`
    <ul>
      ${items}
    </ul>
  `
}
```
and remove the `<ul>` `</ul>` elements below on lines 67 and 69.

**Review by mphstudios — CHANGES_REQUESTED** [review]

File: packages/11ty/_includes/components/contributor/bio.js (hunk 2 above)
Comment (geealbers) [reply packages/11ty/_includes/components/contributor/bio.js:55]: @mphstudios, sure, though I'm getting an unexpected identifier error on that code. Seems to be expecting something different in `const contributorPagesList = { ... }`?

```
[11ty] 2. Unexpected identifier (via SyntaxError)
[11ty] Original error stack trace: /Users/galbers/GitHub/quire/packages/11ty/_includes/components/contributor/bio.js:49
[11ty]       const items = pages.map(({ data, url }) => {
[11ty]             ^^^^^
```
One solution might be to leave `contributorPages` as it was, and then add 

```js
const contributorPagesList =
    html`
      <ul>
        ${contributorPages}
     </ul>
   `
```

And then return 

```js
<div class="quire-contributor__details media-content">
  ${contributorImage}
  ${contributorBio}
  ${contributorPagesList}
</div>
```

Comment (mphstudios) [reply packages/11ty/_includes/components/contributor/bio.js:55]: @geealbers I updated/corrected my suggested rewrite of `contributorPages` to use an anonymous function, 

```
const contributorPagesList = () => {
  const items = pages.map(({ data, url }) => {
    const { label, subtitle, title } = data
    const classes = ['quire-contributor__page-link']
    const name = pageTitle({ label, subtitle, title })
    return `<li>${link({ classes, name, url })}</li>`
  })
  return html`
    <ul>
      ${items}
    </ul>
  `
}
```
then
```
<div class="quire-contributor__details media-content">
  ${contributorImage}
  ${contributorBio}
  ${contributorPagesList()}
</div>
```

**Pull request merged**